src/task3/Complex.java

```diff
@@ -0,0 +1,61 @@
+package task3;
+
+public class Complex {
+    private int realNumber;
+    private int imaginaryNumber;
+
+    public Complex() {
+        this.realNumber = 0;
+        this.imaginaryNumber = 0;
+
+    }
+
+    public Complex(int realNumber, int imaginaryNumber) {
+        this.realNumber = realNumber;
+        this.imaginaryNumber = imaginaryNumber;
+    }
+
+    public int Re() {
+        return realNumber;
+    }
+
+    public int Im() {
+        return imaginaryNumber;
+    }
+
+
+    public Complex add(Complex z1) {
+        int realNumber = this.realNumber + z1.realNumber;
+        int imaginaryNumber = this.imaginaryNumber + z1.imaginaryNumber;
+        return new Complex(realNumber, imaginaryNumber);
+
+    }
+
+    public Complex substract(Complex z1) {
+        int realNumber = this.realNumber - z1.realNumber;
+        int imaginaryNumber = this.imaginaryNumber - z1.imaginaryNumber;
+        return new Complex(realNumber, imaginaryNumber);
+
+    }
+
+    public void printComplex() {
```

Comment:
I like that you identified that the cases are mutually exclusive, but I would like that you use `else if` anyway.